crates/bevy_internal/src/default_plugins.rs

```diff
@@ -1,144 +1,69 @@
-        }
-
+        bevy_winit:::WinitPlugin,
         #[cfg(feature = "bevy_render")]
-        {
-            group = group
-                .add(bevy_render::RenderPlugin::default())
-                // NOTE: Load this after renderer initialization so that it knows about the supported
-                // compressed texture formats
-                .add(bevy_render::texture::ImagePlugin::default());
-
-            #[cfg(all(not(target_arch = "wasm32"), feature = "multi-threaded"))]
-            {
-                group = group.add(bevy_render::pipelined_rendering::PipelinedRenderingPlugin);
-            }
-        }
-
+        bevy_render:::RenderPlugin,
+        // NOTE: Load this after renderer initialization so that it knows about the supported
+        // compressed texture formats
+        #[cfg(feature = "bevy_render")]
+        bevy_render::texture:::ImagePlugin,
+        #[cfg(feature = "bevy_render")]
+        #[custom(cfg(all(not(target_arch = "wasm32"), feature = "multi-threaded")))]
+        bevy_render::pipelined_rendering:::PipelinedRenderingPlugin,
         #[cfg(feature = "bevy_core_pipeline")]
-        {
-            group = group.add(bevy_core_pipeline::CorePipelinePlugin);
-        }
-
+        bevy_core_pipeline:::CorePipelinePlugin,
         #[cfg(feature = "bevy_sprite")]
-        {
-            group = group.add(bevy_sprite::SpritePlugin);
-        }
-
+        bevy_sprite:::SpritePlugin,
         #[cfg(feature = "bevy_text")]
-        {
-            group = group.add(bevy_text::TextPlugin);
-        }
-
+        bevy_text:::TextPlugin,
         #[cfg(feature = "bevy_ui")]
-        {
-            group = group.add(bevy_ui::UiPlugin);
-        }
-
-        #[cfg(feature = "bevy_pbr")]
-        {
-            group = group.add(bevy_pbr::PbrPlugin::default());
-        }
-
+        bevy_ui:::UiPlugin,
         // NOTE: Load this after renderer initialization so that it knows about the supported
         // compressed texture formats
+        #[cfg(feature = "bevy_pbr")]
+        bevy_pbr:::PbrPlugin,
         #[cfg(feature = "bevy_gltf")]
-        {
-            group = group.add(bevy_gltf::GltfPlugin::default());
-        }
-
+        bevy_gltf:::GltfPlugin,
         #[cfg(feature = "bevy_audio")]
-        {
-            group = group.add(bevy_audio::AudioPlugin::default());
-        }
-
+        bevy_audio:::AudioPlugin,
         #[cfg(feature = "bevy_gilrs")]
-        {
-            group = group.add(bevy_gilrs::GilrsPlugin);
-        }
-
+        bevy_gilrs:::GilrsPlugin,
         #[cfg(feature = "bevy_animation")]
-        {
-            group = group.add(bevy_animation::AnimationPlugin);
-        }
-
+        bevy_animation:::AnimationPlugin,
         #[cfg(feature = "bevy_gizmos")]
-        {
-            group = group.add(bevy_gizmos::GizmoPlugin);
-        }
-
-        group = group.add(IgnoreAmbiguitiesPlugin);
-
-        group
+        bevy_gizmos:::GizmoPlugin,
+        :IgnoreAmbiguitiesPlugin,
```

Comment:
This shouldn't be added to documentation.

Reply:
I agree but I couldn't find a way to exclude this from the documentation via the macro. (macros are hard)

Reply:
I'm not experienced with macros, but maybe we can make it respect `#[doc(hidden)]`?

Reply:
Well I would need to check if `#[doc(hidden)]` is present and then hide the documentation entry.

Reply:
Fixed in https://github.com/bevyengine/bevy/pull/11460/commits/ab5ff9774d368389d72ef36225ace8129184a80d